rules/macos/execution_script_via_automator_workflows.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2020/12/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the execution of Automator Workflows process followed by a network connection from it's XPC service. Adversaries may drop a custom workflow template that host
+malicious JXA code as an alternative to using osascript.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious Workflows Automator Execution"
```

```suggestion
name = "Suspicious Automator Workflows Execution"
```

Is this the term that's more commonly used?